File: WebApp/Scripts/app/model/ERROR_LOG.js

Ext.define('WEBAPP.model.ERROR_LOG', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'LOGTIME', type: 'string' },
        { name: 'PG', type: 'string' },
        { name: 'MSG', type: 'string' },
        { name: 'USERID', type: 'string' }
    ]
});

File: WebApp/Models/ERROR_LOG.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class ERROR_LOG : JCLib.Mvc.BaseModel
    {
        public string LOGTIME { get; set; }
        public string PG { get; set; }
        public string MSG { get; set; }
        public string USERID { get; set; }
    }
}